charger/tasmota.go

```diff
@@ -33,34 +32,40 @@ func NewTasmotaFromConfig(other map[string]interface{}) (api.Charger, error) {
 		Password     string
 		StandbyPower float64
 		Channel      int
```

Comment:
```suggestion
		Channel      []int
```

mapstructure erlaubt dann Wert als auch Liste

Reply:
Hallo @andig , danke fürs drauf schauen. Das ist mir schon klar, es geht aber darum warum wir dem `channels` (nicht `channel`), Parameter über die Template yaml Spezifikation, keine Liste mit eckigen Klammern und kommasepariert übergeben können.
Ich hatte die eckigen Klammern in die `render `Statements eingebaut,  wenn man dann im Template bspw. für `channels` 1,2,3 definiert, wird die Liste OHNE Kommas generiert.

Reply:
Hallo @andig ,

mit der folgenden Meter Template yaml, bekomme ich einen Fehler, wenn ich bei der Eingabe des Wertes des `channels` Parameter eine Lise definiere:

```yaml
template: tasmota
products:
  - description:
      generic: Tasmota
group: switchsockets
params:
  - name: usage
    choice: ["grid", "pv", "battery"]
  - name: host
  - name: user
    required: false
    help:
      de: Standard-User ist admin
      en: admin is default
  - name: password
    required: false
    mask: true
  - name: channels
    default: 1
    required: true
    description:
      de: Liste der Meterkanäle
      en: Meter channel number list
    help:
      de: Liste der Meterkanäle [1,2,..,8], bei Geräten mit mehr als einem Meter
      en: List of meter channel numbers [1,2,....,8]
render: |
  type: tasmota
  uri: http://{{ .host }}
  usage: {{ .usage }}
  user: {{ .user }}
  password: {{ .password }}
  channels: {{ .channels }}  # list of meter channels [1,2,....,8]
```

Fehler bei der Konfiguration des Templates mit `evcc configure`:
Der Fehler kommt m.E. schon beim Rendern des Templates.
```
Wähle eine der folgenden Gerätekategorien aus:

 Wähle eine der folgenden Gerätekategorien aus: PV Wechselrichter (oder entsprechender Stromzähler)

- Konfiguration PV Wechselrichter (oder entsprechender Stromzähler)

 Wähle einen PV Wechselrichter (oder entsprechender Stromzähler): Tasmota [Schaltbare Steckdosen]

Führe folgende Einstellungen durch:

 IP-Adresse oder Hostname 192.168.178.95
 Benutzerkonto 
 Passwort [? for help] 
 Liste der Meterkanäle [1,2]

Teste die pv Konfiguration von Tasmota [Schaltbare Steckdosen] ...
   Fehler: cannot create meter 'tasmota': 1 error(s) decoding:

* cannot parse 'Channels[0]' as int: strconv.ParseInt: parsing "[1,2]": invalid syntax

 Der Test der pv Konfiguration von Tasmota [Schaltbare Steckdosen] ist fehlgeschlagen. Soll pv trotzdem in die Konfiguration aufgenommen werden? No
```

Reply:
Die Params haben Typen. Default ist string. Es muss hier valuelist list (Name nicht sehr intuitiv)

Reply:
Die gültigen Werte sind hier: https://github.com/evcc-io/evcc/blob/03da1f129f37648ac8ccf9ac008fbc143f345274/util/templates/paramtype.go. Richtig ist `stringlist`, sorry.